### PR TITLE
new Update main.py

### PR DESCRIPTION
**Pull request by wanda-carlson**
new body

### CODE REVIEW

**Comment by wanda-carlson** [comment]
Test comment

**Comment by wanda-carlson** [comment]
slack comment

File: src/main.py [hunk 1]
```diff
@@ -3,3 +3,4 @@
 """
 if __name__ == "__main__":
     print("Hello world")
+    print("Hello world")
```
Comment (henry-richard) [inline src/main.py:6]:
```suggestion
    print("Hello worlds")
```

**Review by henry-richard — CHANGES_REQUESTED** [review]

File: src/main.py [hunk 2]
```diff
@@ -3,3 +3,4 @@
 """
 if __name__ == "__main__":
     print("Hello world")
+    print("Hello worlds")
```
Comment (sarah-paidpaper) [inline src/main.py:6]:
```suggestion
    print("Hello worldssss")
```

File: src/main.py [hunk 3]
```diff
@@ -3,3 +3,4 @@
 """
 if __name__ == "__main__":
     print("Hello world")
+    print("Hello worldssss")
```
Comment (wanda-carlson) [inline src/main.py:6]:
```suggestion
    print("Hello worldsssss")
```

File: src/main.py [hunk 4]
```diff
@@ -3,3 +3,4 @@
 """
 if __name__ == "__main__":
     print("Hello world")
```
Comment (henry-richard) [inline src/main.py:5]:
```suggestion
    print("Hello worlds")
```

**Review by henry-richard — APPROVED** [review]

**Comment by wanda-carlson** [comment]
**Pullflow** add review comment "wow"

**Review by wanda-carlson — COMMENTED** [review]
message

**Review by wanda-carlson — COMMENTED** [review]
Wow

**Review by sarah-paidpaper — APPROVED** [review]

**Comment by henry-richard** [comment]
aiogheiosgse

**Comment by henry-richard** [comment]
```
Hey 
no Yes
```

**Comment by henry-richard** [comment]
```
Hey 
no Yes
```

**Comment by henry-richard** [comment]
`gdlksfns oei`
`alsnfoienf`
`afnseof`

**Pull request merged**